buildkite/access_tokens_test.go

```diff
@@ -0,0 +1,56 @@
+package buildkite
+
+import (
+	"fmt"
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+func TestAccessTokensService_Get(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/v2/access-token", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		fmt.Fprint(w, `{"uuid": "b63254c0-3271-4a98-8270-7cfbd6c2f14e","scopes": ["read_build"]}`)
+	})
+
+	ats, _, err := client.AccessTokens.Get()
+	if err != nil {
+		t.Errorf("AccessTokens.Get returned error: %v", err)
+	}
+
+	fmt.Print(ats)
+
+	want := &AccessToken{
+		UUID:   String("b63254c0-3271-4a98-8270-7cfbd6c2f14e"),
+		Scopes: &[]string{"read_build"},
+	}
+	if !reflect.DeepEqual(ats, want) {
+		t.Errorf("AccessTokens.Get returned %+v, want %+v", ats, want)
+	}
+}
+
+func TestAccessTokensService_Revoke(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/v2/access-token", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "DELETE")
+		fmt.Fprint(w, `{}`)
```

Comment:
As mentioned in the comment, I think this status is a StatusNoContent.

I haven't tested this but I think you could return something like:
```suggestion
		w.WriteHeader(http.StatusNoContent)
```

Reply:
Have done this and tweaked the test to not touch body as it was causing EOF